Magento_Theme/layout/default.xml

```diff
@@ -6,5 +6,14 @@
                 <argument name="logo_width" xsi:type="number">185</argument>
             </arguments>
         </referenceBlock>
+
+        <referenceContainer name="footer">
+            <block class="Magento\Framework\View\Element\Template" name="custom.footer" template="Magento_Theme::custom_footer.phtml">
```

Comment:
```
Magento\Framework\View\Element\Template
```
is the default class for all the blocks created in XML by the directive `block`. if you need `Template`, so, just don't specify `class`, that's unnecessary. Use `class` only when you have to set another one, not `Template`